onnx/backend/test/cpp/driver_test.cc

```diff
@@ -322,6 +323,17 @@ class ONNXCppDriverTest : public ::testing::TestWithParam<
                 true);
           }
         }
+/*
+ * Examine functions in onnxifi_ext
+ */
+#ifdef ONNXIFI_ENABLE_EXT
+        onnxExtensionFunctionPointer* f;
+        ASSERT_TRUE(IsGtestAssertMemorySafeSuccess(
```

Comment:
why not test `onnxSetIOAndRunGraph` function extension as part of the driver_test (given that the function is available)

Reply:
That's exactly I would like to do. But the thing is since the driver test is down, we cannot test it anyway. Since I dont want to cause some potential bugs here, I will add related test in test driver once we fixed onnxifi driver test.